### ISSUE TITLE
Links auf der Teamseite

### ISSUE THREAD

**Issue by motrellin** [FLBadoras/wiki#92]
Mir fällt auf, dass die Überschriften (aka Teammitgliedsname + Rang) nicht richtig verlinken, sondern immer nur auf die H1-Überschrift der Teamseite verweisen. Da könnte man mal Nachforschungen anstellen.

**Comment by motrellin** [comment]
Zusätzlicher Schönheitsfehler, der damit vermutlich zusammenhängt: Wenn man z.B. nach Rubior sucht, wird das zwar korrekt vorgeschlagen, jedoch führt der Link auch nur zum Beginn der Seite. Zum Vergleich: Sucht man nach "Funktion", findet man bspw. diese [Seite](https://wiki.badoras.life/systems/drawing/#funktion). Man beachte die URL.

**Comment by RobZenFL** [comment]
Ist das noch aktuell? Habs grad mal probiert und hatte keine Probleme.

**Comment by motrellin** [comment]
> Ist das noch aktuell? Habs grad mal probiert und hatte keine Probleme.

Ja. Problem macht das nicht, aber es ist ein Schönheitsfehler. Beachte die URL's bei beiden Beispielen.
